terraform/app_service.tf

```diff
@@ -115,6 +115,7 @@ resource "azurerm_linux_web_app_slot" "dev" {
     ignore_changes = [app_settings, tags]
   }
 
+  # fixtures
```

Comment:
Not a huge deal but this comment is going to be out of date as soon as #814 and #827 are merged